cross/x264/Makefile

```diff
@@ -19,7 +19,6 @@ GNU_CONFIGURE = 1
 CONFIGURE_ARGS  = --enable-shared
 CONFIGURE_ARGS += --enable-pic
 CONFIGURE_ARGS += --disable-opencl
-CONFIGURE_ARGS += --prefix=$(INSTALL_PREFIX)
```

Comment:
why is the prefix being removed?  Presumably because the spksrc.configure.mk adds it anyway?